docs/server.rst

```diff
@@ -115,6 +115,13 @@ Also the following Debian packages are required:
 * ``python-dev``
 * ``libevent-dev``
 
+Debian 10 required following packages:
```

Comment:
If we give a distro here, we should also give a distro above -- or replace the above list.